plat/hisilicon/hikey/hikey_security.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2018, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <assert.h>
+#include <debug.h>
+#include "hikey_private.h"
```

Comment:
@petegriffin 
Can you move this 'local' include to be after the system includes.

Reply:
@davidcunado-arm sure, I just pushed a new patch